uniffi/src/interface/mod.rs

```diff
@@ -485,9 +560,6 @@ impl Enum {
 
 impl APIConverter<Enum> for weedle::EnumDefinition<'_> {
     fn convert(&self, _ci: &ComponentInterface) -> Result<Enum> {
-        if self.attributes.is_some() {
```

Comment:
We do the attributes parsing beforehand for this. (since the enum might be an error)